build.gradle.kts

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Facebook, Inc. and its affiliates.
+
+// This source code is licensed under the MIT license found in the
+// LICENSE file in the root directory of this source tree.
+
+buildscript {
```

Comment:
This is the cool part - because the build script is in kotlin (which Android Studio recognizes just fine) but is in the module root, you can build your module just fine for testing - with specific versions - but when people import it into their work project, your project definition doesn't really exist. They import it using their project's gradle versions.